packages/flutter/lib/src/gestures/binding.dart

```diff
@@ -130,12 +130,21 @@ mixin GestureBinding on BindingBase implements HitTestable, HitTestDispatcher, H
     } else if (event is PointerUpEvent || event is PointerCancelEvent) {
       result = _hitTests.remove(event.pointer);
     } else if (event.down) {
+      // In addition to PointerDownEvents, this block also handles
+      // PointerMoveEvents. Because PointerMoveEvents should be dispatched to
+      // the same place that their initial PointerDownEvent was, we want to
+      // re-use the path we found when the pointer went down, rather than do hit
+      // detection each time we get a PointerMoveEvent.
       result = _hitTests[event.pointer];
-    } else {
-      return; // We currently ignore add, remove, and hover move events.
     }
-    if (result != null)
+    assert(() {
+      if (debugPrintMouseHoverEvents && event is PointerHoverEvent)
+        debugPrint('$event');
+      return true;
+    }());
```

Comment:
do we expect this to be useful long term?

Reply:
Yes, I do think that people building interfaces will occasionally want to see the hover events as they occur.